docs/policy.md

```diff
@@ -13,6 +13,10 @@ A policy has following properties.
   you can specify target resource using "path" and "properties"
 - condition : additional condition (see below)
 - tenant_id : regexp matching the tenant, defaults to ``.*``
+- is_domain_owner: type of the token's scope.
```

Comment:
@piotr-dulikowski As @p-kozlowski commented in point 6, I also expected that `is_domain_owner` is a condition keyword like `is_owner`.

Can you change the design? Or, if you think it's necessary to place `is_domain_owner` at the top level of the policy, can you elaborate on the reason?

Reply:
@morrisson,
Semantics of `is_owner` are not similar to `is_domain_owner`.
- Properties like `principal`, `tenant_id`/`tenant_name`, `is_domain_owner` determine _if given policy is used_. This logic can be found in [policy.match](https://github.com/cloudwan/gohan/pull/686/files#diff-4166c24150929a3ed94d148ea2a87b80R655).
- The `condition` property defines _visibility_ of resources. If a given policy is decided to be used, `is_owner` may deny access to a resource, but it will not cause a different policy to be used instead. Note that, depending on the token scope, `is_owner` may filter resources either:
	- by both tenant_id and domain_id (in case of scoping to project),
	- only by domain_id (in case of scoping to domain),
	- not at all (in case of scoping to admin project).